crawler.js

```diff
@@ -460,13 +460,17 @@ export class Crawler {
     }
   }
 
-  async workerDone(workerid) {
+  async workerEmpty(workerid) {
     if (this.screencaster) {
       //logger.debug("End Screencast", {workerid}, "screencast");
       await this.screencaster.stopById(workerid, true);
     }
   }
 
+  async workerDone(/*workerid*/) {
+    // track worker done
+  }
```

Comment:
That's a good point, I kept this after adding workerEmpty() thinking this may be used to add a status update later, but, thinking more about it, a worker done status doesn't make sense separate from full crawl done:
either a worker is empty/idle waiting for more work, while at least one other is working, or all workers/crawl is done..